sst.config.ts

```diff
@@ -81,6 +88,8 @@ const runDockerCompose = async () => {
 
 export const sstConfig: SSTConfig = {
   async config(_input) {
+    const { app } = _input as any
```

Comment:
explicit disable of eslint any?

Reply:
This is the best I can do

Reply:
Oh, I was just suggesting to add an eslint-disable-next-line rule to avoid linting errors/warnings